hstool/templates/tool/relation_add.html

```diff
@@ -43,172 +64,190 @@
+			</p>
+			<a class="btn btn-primary" href="{% url 'drivers:add' %}" target="_blank">
+			  Add new driver of change
+			</a>
+		  </div>
+		</div>
+
+		<div data-show-if="show-figureindicators">
+		  <div class="form-group">
+			<div class="col-sm-6">
+			  {% render_field form.figureindicator class='form-control' %}
+			  {{ form.figureindicator.errors }}
+			</div>
+		  </div>
+		  <div class="form-group">
+			<p class="col-sm-4">
+			  Is your indicator, fact or figure missing from the list?
+			</p>
+			<a class="btn btn-primary" href="{% url 'figures:add' %}" target="_blank">
+			  Add new indicator, fact and figure
+			</a>
+		  </div>
+		</div>
+
+		<div data-show-if="show-impacts">
+		  <div class="form-group">
+			<div class="col-sm-6">
+			  {% render_field form.impact class='form-control' %}
+			  {{ form.impact.errors }}
+			</div>
+		  </div>
+		  <div class="form-group">
+			<p class="col-sm-4">
+			  Is your impact missing from the list?
+			</p>
+			<a class="btn btn-primary" href="{% url 'impacts:add' %}" target="_blank">
+			  Add new impact
+			</a>
+		  </div>
+		</div>
+
+	  </fieldset>
+
+		<div data-show-if="show-drivers">
+			<fieldset class="well">
+				<div class="form-group">
+					<label class="col-sm-6">{{ form.relationship_type|label }}
+					</label>
+				</div>
+
+				<div class="form-group">
+					<div class="col-sm-6">
+					{% render_field form.relationship_type class='form-control' %}
+					{{ form.relationship_type.errors }}
+					</div>
+				</div>
+				  <div class="form-group">
+					  <label class="col-sm-6"> {{ form.description|label }}
+						</label>
+				  </div>
+
+				<div class="form-group">
+				  <div class="col-sm-6">
+					{% render_field form.description class='form-control' %}
+					{{ form.description.errors}}
+				  </div>
+				</div>
+			 </fieldset>
+		</div>
+
+	   <fieldset class="well">
+		<div class="form-group">
+		  <label class="col-sm-6">{{ form.figureindicators|label }}
+		  </label>
+		</div>
+		<div class="form-group">
+		  <div class="col-sm-6">
+			{% render_field form.figureindicators class='form-control' %}
+			{{ form.figureindicators.errors}}
+		  </div>
+		</div>
+
+		<div class="form-group">
+		  <p class="col-sm-4">
+			Is your indicator, fact or figure missing from the list?
+		  </p>
+		  <div class="col-sm-2">
+			<a class="btn btn-primary" href="{% url 'figures:add' %}" target="_blank">
+			  <span class="glyphicon glyphicon-plus"></span> &nbsp;
+			  Add new indicator, fact or figure
+			</a>
+		  </div>
+		</div>
+	  </fieldset>
+
+	  <div class="form-actions">
+		<div class="col-sm-offset-2">
+		  <button type="submit" name="draft" value="false"
+				  class="btn btn-primary">Publish</button>
+		  <button type="submit" name="draft" value="true"
+				  class="btn btn-default">Save as draft</button>
+		  <a href="{% url 'assessments:detail' assessment.id %}"
+			 class="btn btn-link" role="button">Cancel</a>
+		</div>
+	  </div>
+	</form>
```

Comment:
Fix indentation. You should use 2 spaces indentation for html/jinja templates. Also for javascript files.